platform/app/Home.module.css

```diff
@@ -21,6 +21,12 @@ section.details {
   margin-bottom: 80px;
 }
 
+section.innerSections {
```

Comment:
a new class name creates more boilerplate than necessary; can be replaced with `section.details > section`